addOns/accessControl/src/main/java/org/zaproxy/zap/extension/accessControl/view/AccessControlStatusPanel.java

```diff
@@ -121,7 +125,7 @@ protected Component getWorkPanel() {
             workPane = new JScrollPane();
             workPane.setName("AccessControlResultsPane");
             workPane.setViewportView(getScanResultsTable());
-            workPane.setFont(new java.awt.Font("Dialog", java.awt.Font.PLAIN, 11));
+            workPane.setFont(FontUtils.getFont(FontType.workPanels));
```

Comment:
Should be removed instead, this is a status panel (despite the method name), setting a font is not needed here either way (same for the table).